docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/azure-event-hubs-source.md

```diff
@@ -116,6 +116,8 @@ When Sumo Logic detects an issue it is tracked by Health Events. The following t
 | ThirdPartyGeneric | Normally due to an error communicating with the third party service APIs. | Yes | The Source will retry for up to 90 minutes, after which retries will be attempted every 60 minutes. | ThirdPartyGenericError |
 | FirstPartyGeneric | Normally due to an error communicating with the internal Sumo Logic APIs. | Yes | The Source will retry for up to 90 minutes, after which retries will be attempted every 60 minutes. | FirstPartyGenericError |
 
+** You may receive an error: `At least one receiver for the endpoint is created with epoch of '0', and so non-epoch receiver is not allowed. Either reconnect with a higher epoch, or make sure all epoch receivers are closed or disconnected.` This indicates Sumo Logic cannot pull from the even hub because it detects another solution pulling from it. See [this Microsoft documentation](https://techcommunity.microsoft.com/t5/azure-paas-blog/eventhub-the-behavior-of-eph-sdk-behavior-of-epoch/ba-p/3888275) for more information. You'll need to create a new event hub instance where Sumo Logic is the primary consumer. 
```

Comment:
```suggestion
** You may receive an error: `At least one receiver for the endpoint is created with epoch of '0', and so non-epoch receiver is not allowed. Either reconnect with a higher epoch, or make sure all epoch receivers are closed or disconnected.` This indicates Sumo Logic cannot pull from the event hub because it detects another solution pulling from it. See [this Microsoft documentation](https://techcommunity.microsoft.com/t5/azure-paas-blog/eventhub-the-behavior-of-eph-sdk-behavior-of-epoch/ba-p/3888275) for more information. You'll need to create a new event hub instance where Sumo Logic is the primary consumer. 
```

Reply:
Also, does this need to be bulleted or can we remove?